spec/models/user_spec.rb

```diff
@@ -265,4 +265,26 @@
       expect(user).to_not have_admin_access
     end
   end
+
+  describe '#first_and_last_name' do
```

Comment:
When @adbatista claimed about the use of the subject he was referring to something like this:
```ruby
describe '#first_and_last_name' do
  subject(:user) { build_stubbed(:user, name: name) } # Setup step
  
  context 'when name has 3 parts' do
    let(:name) { 'Name1 Name2 Name3' }
    
    it 'returns only first and last name' do
      expect(user.first_and_last_name).to eq 'Name1 Name3' # Verify step
    end
  end
end
```